src/pygambit/game.pxi

```diff
@@ -336,6 +336,39 @@ class Game:
         g.title = title
         return g
 
+    def to_arrays(self) -> typing.List[np.array]:
+        """Reverse function to from_arrays:
+
+        To_arrays, for a given game, generates players’ payoff tables represented as numpy arrays.
```

Comment:
Don't need to repeat the function name, or to say it's for a given game, those are obvious from the definition of the function.  Can just say "Generates the payoff tables for players represented as numpy arrays."